ui/src/alerting/components/notifications/SMTPMessage.tsx

```diff
@@ -0,0 +1,43 @@
+// Libraries
+import React, {FC, ChangeEvent} from 'react'
+
+// Components
+import {Form, Input, TextArea} from '@influxdata/clockface'
+
+interface Props {
+  to: string
+  subjectTemplate
+  bodyTemplate: string
+  onChange: (e: ChangeEvent) => void
+}
+
+const SMTPMessage: FC<Props> = ({
+  to,
+  subjectTemplate,
+  bodyTemplate,
+  onChange,
+}) => {
+  return (
+    <>
+      <Form.Element label="To">
+        <Input value={to} name="to" onChange={onChange} />
+      </Form.Element>
+      <Form.Element label="Subject">
+        <Input
+          value={subjectTemplate}
+          name="subjectTemplate"
+          onChange={onChange}
+        />
+      </Form.Element>
+      <Form.Element label="Body">
+        <TextArea
```

Comment:
soooo many inputs. I don't even want to think about adding validation...

Reply:
that will be an entirely different story